src/Pages/3DPrintingConsole/3D-console.css

```diff
@@ -22,7 +22,19 @@
   color: #61dafb;
 }
 
-#secondRow
-{
-  margin-left: 35px !important;
+.Console-3d-label-1 {
```

Comment:
use only lowercase and dashes for CSS classnames/IDs. the [google style guide](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming) goes in depth for the naming conventions